driver-core/src/main/com/mongodb/client/model/changestream/ChangeStreamDocument.java

```diff
@@ -58,6 +59,8 @@
     private final BsonInt64 txnNumber;
     private final BsonDocument lsid;
     private final BsonDateTime wallTime;
+    @BsonExtraElements
```

Comment:
There are a couple of annotations that aren't allowed to target constructor parameters.  I'm not sure why, but that's why this annotation is here rather than on the constructor

Reply:
Feel free to fix or add a ticket so they can be.